src/index.html

```diff
@@ -6,8 +6,45 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page">
+    <div class="card" data-qa="card">
+
+      <a href="#" class="card__link link">
```

Comment:
```suggestion
    <div class="card" data-qa="card">
      <a href="#" class="card__link link">
```
Don't leave empty lines between parent and child elements